src/_common/community/channel/channel-permissions.ts

```diff
@@ -0,0 +1,29 @@
+export class ChannelPermissions {
+	public static readonly ACTION_POSTING = 'posting';
```

Comment:
public by default

Reply:
I think we should export consts outside of the class, though.
```
export const COMMUNITY_CHANNEL_PERMISSIONS_ACTION_POSTING = 'posting';
```
I know it's long in this case, but it allows components to use the const without having to import the whole model class.